src/index.js

```diff
@@ -178,6 +182,10 @@ function isJSON(contentType) {
   return contains('application/json')(contentType);
 }
 
+function isPDF(contentType) {
+  return contains('application/pdf')(contentType);
+}
```

Comment:
When the content-type is application/pdf our http response are neither a json or plain text, the response are a binary file so fetch api need to call the method blob